go-controller/pkg/ovn/policy_old.go

```diff
@@ -855,14 +910,19 @@ func (oc *Controller) addNetworkPolicyOld(policy *knet.NetworkPolicy) {
 			if fromJSON.IPBlock != nil {
 				ingress.addIPBlock(fromJSON.IPBlock)
 			}
-			if fromJSON.NamespaceSelector != nil {
+
+			if fromJSON.NamespaceSelector != nil && fromJSON.PodSelector != nil {
+
```

Comment:
Can you please add a comment here to explain why we handle this case similar to the else blocks that already have comments.